src/relaxations/sparse/Chordal.jl

```diff
@@ -1,76 +1,58 @@
+using CliqueTrees, Graphs
+using CliqueTrees: EliminationAlgorithm
```

Comment:
Graphs is already imported in the module; please move the import to `src/relaxations/Relaxation.jl` so that they are all together, and keep it an import. It is used only for a few identifiers in this file, so I don't want to have all the exports in the current namespace.